technologies/job/python/python-3.9-base/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM python:3.9.7-slim-bullseye
+
+ENV DEBIAN_FRONTEND noninteractive
+
+ENV LC_ALL C
+ENV LD_RUN_PATH /usr/local/lib
+
+# LIGHT DEPENDENCIES START
+RUN apt-get update -qq && \
+    apt-get install -yqq --no-install-recommends \
+      ftp wget curl unzip telnet openssh-client git apt-transport-https ca-certificates \
+      krb5-user libpam-krb5 libpam-ccreds locales openjdk-11-jdk ca-certificates-java ant && \
```

Comment:
Jdk11 and hadop 2.6.5 is not officially supported. You can confirm that it works?